app/src/main/java/com/geeksville/mesh/ui/NodeDetail.kt

```diff
@@ -196,7 +214,51 @@ private fun NodeDetailRow(label: String, icon: ImageVector, value: String) {
 }
 
 @Composable
-private fun NodeDetailsContent(node: NodeEntity) {
+private fun DeviceDetailsContent(
+    node: NodeEntity,
+) {
+    val context = LocalContext.current
+    val deviceHardware =
+        DeviceHardware.getDeviceHardwareFromHardwareModel(context, node.user.hwModel)
```

Comment:
this can move to the ViewModel and use app context. also needs some exception handling.